docker-compose.yml

```diff
@@ -2,11 +2,11 @@
 
 version: '3.0'
 services:
-    node:
-        build:
-            context: .
-            dockerfile: docker/Dockerfile
+    digbot:
+        build: .
+        volumes:
+        # Links the local.json config to the container for local development
```

Comment:
embedding here

Reply:
Embedding of some options in inconsistent across our docker compose files, specifically, docker-compose-test.yml has 4 space embedding for lines 11 and 12 for volumes: values but this file does not